plugins/generator-1.14.4/forge-1.14.4/procedures/entity_heal.java.ftl

```diff
@@ -0,0 +1 @@
+if(${input$entity} instanceof LivingEntity)((LivingEntity) ${input$entity}).setHealth(((${input$entity} instanceof LivingEntity)?((LivingEntity) ${input$entity}).getHealth():-1) + ${input$amount}f);
```

Comment:
no need to do the extra instanceof check here

Reply:
extra instanceof has been removed